src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -85,6 +89,23 @@ public async Task ExecuteAsync(string stackName, UserDeploymentSettings? userDep
             await CreateDeploymentBundle(orchestrator, selectedRecommendation, cloudApplication);
 
             await orchestrator.DeployRecommendation(cloudApplication, selectedRecommendation);
+
+            var displayedResources = await _displayedResourcesHandler.GetDeploymentOutputs(cloudApplication, selectedRecommendation);
+            DisplayOutputResources(displayedResources);
+        }
+
+        private void DisplayOutputResources(List<DisplayedResourceItem> displayedResourceItems)
+        {
+            _toolInteractiveService.WriteLine("Created AWS Resources");
+            _toolInteractiveService.WriteLine("---------------------");
+            foreach (var resource in displayedResourceItems)
+            {
+                _toolInteractiveService.WriteLine($"{resource.Description}:");
+                foreach (var resourceKey in resource.Data.Keys)

```

Comment:
There should be separate write lines for id and type along with description. I have further comment that we should not duplicate id and type in the data collection. Even for resources like beanstalk environments that we have a special handler it is still worth putting the id and type out so user can find the resources in the AWS console.

Reply:
Fixed